napari_skimage_regionprops/_table.py

```diff
@@ -29,10 +29,19 @@ def __init__(self, layer: "napari.layers.Layer", viewer: "napari.Viewer" = None)
 
         self._view = QTableWidget()
         self._view.setEditTriggers(QTableWidget.EditTrigger.NoEditTriggers)
+
         if hasattr(layer, "properties"):
-            self.set_content(layer.properties)
+            content = layer.properties
+        elif hasattr(layer, "features"):
+            content = layer.features.to_dict('list')
+
+        # to accomodate passing features to surfaces through the metadata
+        elif 'features' in layer.metadata.keys():
```

Comment:
Can you give an example (plugin) where there is data in `layer.metadata['features']` but not in `layer.features`? Might it be possible to talk to the developers of these plugins so that they use `layer.features`? It would make more plugjns compatible to each other if we all use the same API 😉